### PR TITLE
Debug

### PR DESCRIPTION
**Pull request by Jonathan-dehaay**
Corrected lot's of issues.

### CODE REVIEW

File: Controllers/DetailedInterventionViewController+Weather.swift [hunk 1]
```diff
@@ -40,29 +40,33 @@ extension AddInterventionViewController {
...
   }
 
   func checkTemperatureTextField() {
     let temperature = temperatureTextField.text!
 
     if temperature.count > 0 {
-      if negativeTemperature.titleLabel?.text == "-" && !temperature.contains("-") {
-        temperatureTextField.text = "-" + temperature
+      if negativeTemperature.title(for: .normal) == "-" && !temperatureTextField.text!.contains("-") {
+        temperatureTextField.text = "-" + temperatureTextField.text!
```
Comment (TontonBuck) [inline Controllers/DetailedInterventionViewController+Weather.swift]: There is a method to do that: `insert`

File: Controllers/DetailedInterventionViewController+Weather.swift [hunk 2]
```diff
@@ -40,29 +40,33 @@ extension AddInterventionViewController {
...
+      } else if negativeTemperature.title(for: .normal) == "+" && temperatureTextField.text!.contains("-") {
+        let index = temperatureTextField.text!.index(temperatureTextField.text!.startIndex, offsetBy: 1)
+
+        temperatureTextField.text = String(temperatureTextField.text![index...])
```
Comment (TontonBuck) [inline Controllers/DetailedInterventionViewController+Weather.swift]: You can use `dropFirst()`

File: Controllers/InterventionViewController.swift [hunk 3]
```diff
@@ -551,6 +562,12 @@ class InterventionViewController: UIViewController, UITableViewDelegate, UITable
         interventionButton.frame = CGRect(x: column * width/5.357 + (column + 1) * width/19.737, y: 20 + line * 100, width: 70, height: 70)
         interventionButton.titleEdgeInsets = UIEdgeInsets(top: 100, left: 0, bottom: 0, right: 0)
         interventionButton.addTarget(self, action: #selector(action(sender:)), for: .touchUpInside)
+        interventionButtonsLabels[interventionButton.tag].isHidden = false
+        NSLayoutConstraint.activate([
+          interventionButtonsLabels[interventionButton.tag].topAnchor.constraint(equalTo: interventionButton.bottomAnchor, constant: 5),
+          interventionButtonsLabels[interventionButton.tag].centerXAnchor.constraint(equalTo: interventionButton.centerXAnchor, constant: 0)
```
Comment (TontonBuck) [inline Controllers/InterventionViewController.swift]: You can remove constant argument if you don't use it

File: Clic&Farm-iOS.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -748,7 +746,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 11.4;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.1;
```
Comment (Aquaj) [inline Clic&Farm-iOS.xcodeproj/project.pbxproj]: What does that implies? How much market % are we reaching if we're moving this from 11.4 to 12.1? Also, why exactly are we doing so?

Comment (TontonBuck) [reply Clic&Farm-iOS.xcodeproj/project.pbxproj]: It implies user has to be on iOS 12.1 or above to install the app, this is not what we want. I think deployment targets updated itself when I updated Xcode to 10.1. Deployment target is now default's.

**Pull request merged**